endstate_correction/simulation/amber.py

```diff
@@ -0,0 +1,14 @@
+from openmm.app import AmberPrmtopFile, AmberInpcrdFile
+
+from .base import EndstateCorrectionBase
+
+
+class EndstateCorrectionAMBER(EndstateCorrectionBase):
+    def _get_mm_topology(self) -> AmberPrmtopFile:
+        psf = AmberPrmtopFile(self.top.prm7)
+        return psf
+
+    def _get_mm_coordinate(self) -> AmberInpcrdFile:
+        return AmberInpcrdFile(self.top.rst7)
```

Comment:
I changed the code in the `charmm` and `base` branch of this class to indicate that it is the initial coordinate set with `_get_initial_coordinates`, and also the return type to the actual coordinates with associated units. 

The reason why this does not fail here is because we don't have a test for the Amber files for now.